examples/at_linear_array_transducer/at_linear_array_transducer.py

```diff
@@ -49,11 +49,10 @@ def main():
     else:
         ids = np.arange(1, element_num + 1) - (element_num + 1) / 2
 
-    time_delays = -(np.sqrt((ids * element_pitch) ** 2 + source_focus ** 2) - source_focus) / c0
+    time_delays = -(np.sqrt((ids * element_pitch) ** 2 + source_focus**2) - source_focus) / c0
```

Comment:
Why did it remove the whitespace around second `**` operator while whitespace around the first is left?